### ISSUE TITLE
Example sets $writer->includeColumnHeaders, which has no effect

### ISSUE THREAD

**Issue by asmundstavdahl** [fusonic/spreadsheet-export#9]
The part of the example code that demonstrates Writer sets $writer->includeColumnHeaders to true.
It would be a handy flag to have, but there does not appear to be any code to implement its functionality.

``` php
// Instantiate writer (CSV)
// $writer = new CsvWriter();
// $writer->includeColumnHeaders = true;
// $writer->charset = CsvWriter::CHARSET_ISO;
// Instantiate writer (TSV)
// $writer = new TsvWriter();
// $writer->includeColumnHeaders = true;
// $writer->charset = TsvWriter::CHARSET_ISO;
// Instantiate writer (ODS)
$writer = new OdsWriter();
$writer->includeColumnHeaders = true;
```

**Comment by mburtscher** [comment]
Are you sure? All three writers check for the `$this->includeColumnHeaders` property before printing them.

**Comment by asmundstavdahl** [comment]
For a second there I seriously doubted my grepping skills.

Yeah I see it in the repo. I installed via composer though, and failed to consider that the composer package could be outdated.

It seems the branch used by composer is 0.1.1, which is lagging behind quite a bit (2016-09-20 18:19 UTC).

Would you please consider updating the composer package?

**Comment by asmundstavdahl** [comment]
OK now I found your instructions to use `@dev` at http://fusonic.github.io/spreadsheetexport/#installation.

It would be nice to get a mode updated version with the use of the usual methods, e.g. `composer require fusonic/spreadsheet-export`.

And by the way, the name of the composer package is different from the repo's name apparently; it is "fusonic/spreadsheet**-**export", not "fusonic/spreadsheetexport".
Worth updating http://fusonic.github.io/spreadsheetexport/#installation? :)

```
>> composer require fusonic/spreadsheetexport

                                                                                              
  [InvalidArgumentException]                                                                  
  Could not find package fusonic/spreadsheetexport at any version for your minimum-stability  
   (stable). Check the package spelling or your minimum-stability                             
…

>> composer require fusonic/spreadsheet-export
Using version ^0.1.1 for fusonic/spreadsheet-export
./composer.json has been updated
…
```

**Comment by mburtscher** [comment]
Hmmm you're right. Using `@dev` isn't good practice, we should definitely to a new release which allows a clean version constraint. Since there are breaking changes since the 0.1.1 release it will be a 2.0. Will rename the GitHub repository to "spreadsheet-export" which matches the namespace capitalization and Composer package name.

**Comment by mburtscher** [comment]
Renamed repository to `fusonic/spreadsheet-export` and published version `1.0.0`. So please use version constraint `^1.0` from now on.
